src/bulls_and_cows_lib/board.cpp

```diff
@@ -4,5 +4,41 @@
 namespace bulls_and_cows {
 
     // TODO: define the body of the functions declared in board.cpp
+    // Create a board and with a new random secret code composed of allowed characters
+    Board create_board(const GameOptions& game_options)
+    {
```

Comment:
In the body of this function, you must :
- create a local variable **board** of type **Board**
- modify the data member **board.secret_code.value** so that it contains a sequence of 5 randomly generated characters. (you can use the function **generate_random_character** declared in **input.hpp**)
- return the variable **board**